project/apps/users/src/app/blog-user/blog-user.repository.ts

```diff
@@ -38,4 +39,29 @@ export class BlogUserRepository implements CrudRepositoryInterface<BlogUserEntit
       .findByIdAndUpdate(id, item.toObject(), {new: true})
       .exec();
   }
+
+  public async getSubscribersByUserId(userId: string) {
+    const temp = await this.blogUserModel
+      .aggregate([
+        {
+          $match: {subscribedTo: { has: { userId } }}
+        }
+      ]).exec();
```

Comment:
```suggestion
    .aggregate([
        {
          $match:{ subscribedTo: {$elemMatch: {$eq: userId }} }
        }
      ]).exec();
```

вот так поправильнее